lib/re_captcha/configuration.rb

```diff
@@ -33,7 +36,22 @@ def language_code(locale)
     end
 
     def valid?
-      !private_key.nil? && !public_key.nil?
+      reset_errors
+      set_errors
+      errors.empty?
+    end
```

Comment:
You right it comes with activesupport https://github.com/rails/rails/blob/dac822ef58ae05f0e805222fa8744116080165ac/activemodel/activemodel.gemspec#L21. But they would still be loaded anyway